packages/next-urql/src/Provider.ts

```diff
@@ -59,7 +68,15 @@ export function UrqlProvider({
     React.createElement(
       SSRContext.Provider,
       { value: ssr },
-      React.createElement(DataHydrationContextProvider, { nonce }, children)
+      React.createElement(
+        DataHydrationContextProvider,
+        { nonce },
+        React.createElement(
+          React.Suspense,
```

Comment:
Hm, my only concern here is that there could be legitimate uses for this. I'm imagining that some router systems may use suspense in unexpected ways, so not sure if we can somehow make detection work without a custom boundary?

Reply:
If this Suspense is inserted on the server side, the Promise resolution in the React component will change to streaming instead of waiting.

I'm sorry, but SSR in urql without Suspense will be impossible.

https://next-blog.croud.jp/contents/7m4dShGLIpFkc0GKHIrW

Can you please adjust the server side so that Suspense is not interrupted?